atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/ThreeNodeCassandraCluster.java

```diff
@@ -65,6 +65,16 @@
             .leaders(ImmutableSet.of("localhost"))
             .build());
 
+    private final String cassandraVersion;
+
+    public ThreeNodeCassandraCluster() {
+        this.cassandraVersion = "2.2.8";
```

Comment:
I'm easy on whether it's System.getProperty (you can use `systemProperty` in gradle to set this) or an environment variable that gets passed through.